docs/content/latest/admin/yb-admin.md

```diff
@@ -485,7 +485,10 @@ Sets the preferred availability zones (AZs) and regions.
 
 {{< note title="Note" >}}
 
-When nodes in the the "preferred" availability zones and regions are alive and healthy, the tablet leaders are placed on nodes in those zones and regions. By default, all nodes are eligible to have tablet leaders.
+When nodes in the the "preferred" availability zones and regions are alive and healthy, the tablet leaders are placed on nodes in those zones and regions. 
```

Comment:
@ddorian --

We should add an example here, just like other sections (e.g., see modify_placement_info above).